src/Stratis.Bitcoin.Features.Wallet/Interfaces/IWalletSyncManager.cs

```diff
@@ -22,27 +22,23 @@ public interface IWalletSyncManager
         /// <summary>
         /// Processes a new block.
         /// </summary>
-        /// <param name="block"></param>
         void ProcessBlock(Block block);
 
         /// <summary>
         /// Processes a new transaction which is in a pending state (not included in a block).
         /// </summary>
-        /// <param name="transaction"></param>
```

Comment:
Why did you remove those?

Reply:
They were not completed correctly but I will put them back and complete the `param`.